README.md

```diff
@@ -36,8 +36,9 @@ use({
 ```lua
 -- default config
 local default_config = {
-  disable = {},
+  blacklist = {},
```

Comment:
I would like to use 'blocklist' instead of 'blacklist'.

Reply:
In addition, since the blocked targets are based on file type, I would like to name it `ft_blocklist`.